src/content/learn/tutorial-tic-tac-toe.md

```diff
@@ -899,19 +899,21 @@ body {
 
 ### React Developer Tools {/*react-developer-tools*/}
 
-React DevTools let you check the props and the state of your React components. You can find the React DevTools tab at the bottom of the _browser_ section in CodeSandbox:
+React Developer Tools let you check the props and the state of your React components. It is available as a [Chrome](https://chrome.google.com/webstore/detail/react-developer-tools/fmkadmapgofadopljbjfkapdkoienihi?hl=en), [Firefox](https://addons.mozilla.org/en-US/firefox/addon/react-devtools/), and [Edge](https://microsoftedge.microsoft.com/addons/detail/react-developer-tools/gpphkfbcpidddadnkolkpfckpihlkkil) browser extension. Install it, and the *Components* tab will appear in your browser Developer Tools for sites using React.
 
-![React DevTools in CodeSandbox](../images/tutorial/codesandbox-devtools.png)
+After React Developer Tools installation, you can take the URL present in the _preview_ section of CodeSandbox (similar to "https://***.csb.app/") and open it in a new browser tab.
```

Comment:
I am not seeing the URL, maybe because something changed on CodeSandbox end. We should probably leave a link to `https://yjksdn.csb.app/` here?

Reply:
I wrote *** in the URL as a placeholder for the codesandbox instance ID that the reader will have in their run which could be any random string.
If there's a better way to phrase that the reader needs to use the URL they see in their forked instance, I can change that.

Reply:
Is this identifier generated dynamically? Can we fork this example and generate a stable link to it? There has to be a way to do it in CodeSandbox, I think.

cc @rickhanlonii

Reply:
Yeah that should work?